extensions/wikia/CoppaTool/CoppaToolController.class.php

```diff
@@ -139,7 +139,7 @@ public function renameIPAddress() {
 		}
 
 		$ipAddr = IP::sanitizeIP( $ipAddr );
-		$newIpAddr = '0.0.0.0';
+		$newIpAddr = '::';
```

Comment:
Yeah, that's a good idea. Named constant says what it means. I'll change it. Thanks.